### ISSUE TITLE
Add most popular projects from SciStarter

### ISSUE THREAD

**Issue by punnypenguins** [punnypenguins/projectdelve#7]
Too many projects on [SciStarter](https://scistarter.org/finder) for ALL of them to be added to Project Delve, but I'd like to have the most major/most popular ones on here for people's easy reference so they don't miss them entirely. 

Issue will remain open indefinitely so all SciStarter-related projects can be tagged with it and connected to this issue's history.

**Comment by punnypenguins** [comment]
Just updating for specificity—I decided to narrow this down by only pulling SciStarter projects that can be done online. All of the projects are great! But since Project Delve is meant primarily to recommend internet-accessible citizen science projects that anyone can do from their own homes (or even mobile devices via cellular data if a location doesn't have internet access—read more about that [here](insert link later once I've written the sub-page on Project Delve's website)), so its accessible for all kinds of ages, disabled scientists, school/home/organization settings, etc., I want to make sure I'm not cluttering the site up with less-accessible projects when I've already listed SciStarter as a hub site anyway that people can check out anytime. 

<img width="432" alt="Screen Shot 2021-12-03 at 11 34 52 AM" src="https://user-images.githubusercontent.com/72274725/144647877-76a70f59-e8e1-4fb0-8327-6c81d455282f.png">
